src/clib/lib/include/ert/enkf/enkf_main.hpp

```diff
@@ -89,33 +89,16 @@ int enkf_main_load_from_run_context(enkf_main_type *enkf_main,
                                     enkf_fs_type *sim_fs,
                                     std::vector<run_arg_type *> run_args);
 
-bool enkf_main_case_is_current(const enkf_main_type *enkf_main,
-                               const char *case_path);
-char *enkf_main_read_alloc_current_case_name(const enkf_main_type *enkf_main);
-
-extern "C" void
-enkf_main_init_current_case_from_existing(enkf_main_type *enkf_main,
-                                          enkf_fs_type *source_case_fs,
-                                          int source_report_step);
-
 extern "C" void enkf_main_init_case_from_existing(
     const enkf_main_type *enkf_main, enkf_fs_type *source_case_fs,
     int source_report_step, enkf_fs_type *target_case);
 
 extern "C" bool enkf_main_case_is_initialized(const enkf_main_type *enkf_main,
-                                              const char *case_name);
+                                              const char *case_name,
```

Comment:
The argument list makes a bit less sense now. Not sure what happens when I vary fs. Could you add a doxygen?

Reply:
Connected to the comment below, I thought about this on the way home, and made a lot of sense to move it, so decided to move it in this PR. Put it in a seperate commit, so can revert if the changes are too much.